django/core/fs.py

```diff
@@ -36,7 +36,8 @@ def is_archive(path: str):
 
 def is_image(path: str):
     try:
-        return imghdr.what(path)
+        filetype = imghdr.what(path)
+        return filetype in ['jpeg', 'png', 'gif']
```

Comment:
let's define the accepted image types in one place as a constant (maybe in settings?)

`ACCEPTED_IMAGE_TYPES = ['jpeg', 'png', 'gif']`

we can use it here and in the exception format string that you are building up `library/models.py`